x/ibc/07-tendermint/types/update.go

```diff
@@ -115,7 +115,8 @@ func checkValidity(
 	}
 
 	// assert header height is newer than consensus state
-	if header.GetHeight() <= consState.Height {
+	height := clienttypes.NewHeight(0, header.GetHeight())
```

Comment:
ditto on chainID?

Reply:
Hardcoding with 0 for now will fix in future PR